src/mudlet.h

```diff
@@ -480,6 +485,15 @@ public slots:
     void signal_menuBarVisibilityChanged(const controlsVisibility);
     void signal_toolBarVisibilityChanged(const controlsVisibility);
     void signal_showIconsOnMenusChanged(const Qt::CheckState);
+    // Not currently used but could be by recipient classes for them to execute
+    // retranslateUi(...) and then regenerate all their persistent texts that
+    // are user facing in the new language immediately on locale/language
+    // change instead of waiting until the Mudlet application is restarted with
+    // the new setting.  This would be more efficient than responding to
+    // QEvent::LanguageChange as that event is fired for EVERY invocation of
+    // QCoreApplication::installTranslator and QCoreApplication::removeTranslator
+    // and there can be now FOUR of those every time the GUI language is changed:
+    void signal_guiLanguageChanged(const QString&);
```

Comment:
Let's leave this out - and put it back in when actual live reloading of languages goes in. Given how long the old PR sat unused, I don't have high hopes that it'll be soon, and why have unfinished frameworks sitting around in Mudlet? That'll make code harder to understand.

Reply:
Actually, in a previous comment I'll just spotted somewhere where it can be usefully used - to ensure the "restart Mudlet to complete the language change" message is shown in the **newly selected** language...

Reply:
That's a cool feature - sounds good to do in another PR, not this one not to leak scope.

Reply:
And in the ethos of the Mudlet Event system I'll leave that signal in place to be picked up by that feature in a separate future PR. :stuck_out_tongue_winking_eye: 

... once I have got this one working for me with the CI build images...

Reply:
There's quite a few open from you already... would rather not have half finished things sitting in the main code, doesn't feel tidy you know?

Reply:
Sorry, but it is already now in place! :smiling_imp: And it does work.

Reply:
Ha nice work!